runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/EngineConfigReader.java

```diff
@@ -190,7 +189,7 @@ private boolean validateAnnotatedSchema(
         validate:
         try
         {
-            final JsonObject annotatedSchemaObject = (JsonObject) annotateJsonObject(schemaObject);
+            final JsonObject annotatedSchemaObject = annotator.annotate(schemaObject);
```

Comment:
```suggestion
            final EngineConfigAnnotator() annotator = new EngineConfigAnnotator();;
            final JsonObject annotatedSchemaObject = annotator.annotate(schemaObject);
```

This should be local because it has internal parse state, no need to carry any potential side effects of previous read to next read.